src/components/auth/AuthButton.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../../views/Button/index";
+import { Logout } from "../../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onLogout = this.onLogout.bind(this);
```

Comment:
I think you received feedback on this from @micahoriaso . Let's get rid of this bind statements and the unnecessary constructor. Read on https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56

Reply:
Done it already

Reply:
I can still see bind...I can still see the constructor... what's going on?